docs/setup-indexing.md

```diff
@@ -0,0 +1,55 @@
+# How to add indexing to your Helix 3 project step by step
+
+This document explains how to setup indexing in your Helix 3 project. Currently, indexing into
+an Excel workbook in a SharePoint folder, only, is supported.
+
+## Setting up an initial query index
+
+If you start with the `helix-pages-starter` project you will have a `helix-query.yaml` configuration
+file in your GitHub repository. After having setup your `fstab.yaml` with a mountpoint that points
+into your SharePoint folder, locate a folder where your query index will be located.
+
+Now enter the target URL of that query index location:
+
+1. Select the folder in the Sharepoint UI
+2. Choose action `Copy Link`, and select `Copy` in the popup window
+3. Paste the link into the `helix-query.yaml`s target property
+4. Remove the query string in that link (the question mark `?` and all that follows)
+5. Append the name of your Excel workbook, e.g. `/query-index.xlsx`
+
+When your first page is indexed, the indexer will automatically create a suitable workbook at
+that location.
+
+## Setting up the properties that should be added to the index
+
+In order to decide what properties should be indexed in your HTML pages, look at the existing
+`properties` node in the starter `helix-query.yaml`. Further information on their semantics can be
+found in the [Reference](https://github.com/adobe/helix-index-files#reference) section of the
```

Comment:
isn't this repo going away once helix2 is deprecated?

Reply:
it is 😞 I'll copy its contents to `helix-pages/docs` as well, then

Reply:
@dominique-pfister Actually, the content should be in Google Drive now. I'll give you access.

Reply:
@rofe: should *this* file go to `helix-website/docs` as well?

Reply:
@dominique-pfister yes I think going forward, all documentation should be in `helix-website/docs`.